### ISSUE TITLE
double check node package security reqs

### ISSUE THREAD

**Issue by SaOgaz-NOAA** [nwfsc-fram/boatnet#1900]
Check the yarn.locks for our deployed apps in boatnet to see if they hit the right package versions, right now we need to check:

indirect imports:
http-proxy 1.18.1
node-fetch 2.6.1

Possibly already met:
handlebars 4.5.3
yargs-parser 13.1.2

**Comment by SaOgaz-NOAA** [comment]
@sethgerou-noaa, this yargs-parser dependency chain has top package of @vue/cli-plugin-unit-mocha, which is in the package.json for obs-web and obs-wcgop-optecs. Looks like it's a unit testing package. Have you been actively using it? Can we try updating it?

**Comment by sethgerou-noaa** [comment]
@SaOgaz-NOAA I don't believe we're using Mocha at all in Boatnet (It may have been added by vue-cli during initial project setup) - we can certainly try updating it, or even removing if we're favoring Jest these days.  I personally haven't tried Mocha, but have used Jest a bit and it seems pretty good.

**Comment by SaOgaz-NOAA** [comment]
Ahh, let's try and delete it then and see what happens, I can do that if you want and see if it builds okay, unless you have a strong desire to do it   :D

**Comment by sethgerou-noaa** [comment]
@SaOgaz-NOAA Ha!  No by all means - have at it.
